extra_data/tests/mockdata/base.py

```diff
@@ -139,9 +144,24 @@ def write_metadata(h5file, data_sources, chunksize=16, format_version='0.5'):
 
     if format_version != '0.5':
         h5file['METADATA/dataFormatVersion'] = [format_version.encode('ascii')]
+        now = datetime.utcnow().replace(microsecond=0).strftime('%Y%m%dT%H%M%SZ')
+        h5file['METADATA/creationDate'] = [now.encode('ascii')]
+        h5file['METADATA/daqLibrary'] = [b'1.9.0']
+        h5file['METADATA/karaboFramework'] = [b'2.7.0']
+        h5file.create_dataset('METADATA/proposalNumber', dtype=np.uint32, data=[700000])
+        h5file.create_dataset('METADATA/runNumber', dtype=np.uint32, data=[1])
+        h5file['METADATA/runType'] = [b'Test DAQ']
+        h5file['METADATA/sample'] = [b'No Sample']
+        print(osp.basename(h5file.filename))
+        sequence = int(re.match(r'^(RAW|CORR)\-R\d+\-.*\-S(\d+).h5$', osp.basename(h5file.filename)).groups()[1])
```

Comment:
This assumes that we always use properly formatted names, which is true in conftest, but not if you do `python -m extra_data.tests.make_examples` . I'm ambivalent about whether we correct that or make a fallback to sequence = 0.